Protocol.md

```diff
@@ -206,7 +211,10 @@ Frame Contents
      * (__M__)etadata: Metdadata present
      * (__L__)ease: Will honor LEASE (or not).
      * (__S__)trict: Adhere to strict interpretation of Data and Metadata.
-* __Version__: Version of the protocol.
+* __Version__: Numeric Version of the protocol expressed as two numbers: 
+     * Major Version: 16-bit major version number of the protocol.
```

Comment:
So, I didn't want to add another section of versioning scheme and hence used the heading for version to explain it. Anyways, I will modify it.